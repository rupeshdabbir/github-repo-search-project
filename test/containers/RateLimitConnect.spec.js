import {
  mapStateToProps,
} from 'containers/RateLimit/connect';

describe('RateLimitConnect: mapStateToProps', () => {

  describe('when given the state', () => {
    it.skip('should return relevant properties', () => {
      const state = {
        rateLimit: {
          core: {
            limit: 3,
            remaining: 5,
            reset: 1492972397,
          },
          search: {
            limit: 3,
            remaining: 5,
            reset: 1492972397,
          },
        },
      };
      expect(mapStateToProps(state)).toMatchSnapshot();
    });
  });

});
