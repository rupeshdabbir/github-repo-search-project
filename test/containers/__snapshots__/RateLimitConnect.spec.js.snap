// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RateLimitConnect: mapStateToProps when given the state should return relevant properties 1`] = `
Object {
  "coreLimit": 3,
  "coreRemaining": 5,
  "coreReset": "23-04-2017 19:33",
  "searchLimit": 3,
  "searchRemaining": 5,
  "searchReset": "23-04-2017 19:33",
}
`;
